samples/runtimevar: test doesn't build on go 1.11

```
# gocloud.dev/samples/gocdk-runtimevar [gocloud.dev/samples/gocdk-runtimevar.test]
gocdk-runtimevar/main_test.go:77:51: ee.ExitCode undefined (type *exec.ExitError has no field or method ExitCode)
pass gocloud.dev/samples/gocdk-pubsub (1.11s)
2019/07/10 22:05:28 No test output: "FAIL\tgocloud.dev/samples/gocdk-runtimevar [build failed]"
```

--- a/samples/gocdk-runtimevar/main_test.go
+++ b/samples/gocdk-runtimevar/main_test.go
@@ -18,7 +18,6 @@ import (
 	"context"
 	"errors"
 	"flag"
-	"os/exec"
 	"testing"
 	"time"
 
@@ -72,9 +71,6 @@ func Test(t *testing.T) {
 		cancel()
 		res := <-outc
 		if res.err == context.Canceled {
-			res.err = nil
-		}
-		if ee, ok := res.err.(*exec.ExitError); ok && ee.ExitCode() == -1 { // killed by signal
 			res.err = nil
 		}
 

